CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ option(ENABLE_CUSTOM_EXAMPLES
     "enable private examples in separate directory (applications/custom)" FALSE)
 option(ENABLE_OPENMP "enables OpenMP" FALSE)
 option(ENABLE_PARDISO "enables support for PARDISO solver" FALSE)
-option(ENABLE_PYTHON "create python wrapper for nuto" TRUE)
+option(ENABLE_PYTHON "create python wrapper for nuto" FALSE)
```

Comment:
Why is this turned off?

Reply:
The future python interface is not yet of our PDE development, so I want to save some time in travis and deactivate it. Alternatively, I could have passed `-DENABLE_PYTHON=FALSE` to travis. But since python is currently not working for /nuto/mechanics, I switched it off there...